build/setup.R

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env Rscript
+
+.args <- commandArgs(trailingOnly = TRUE)
+
+if (length(.args) != 1) {
+  stop("Usage: setup.R <flepimop-path>")
+}
+
+# TODO sniff for upgrade mode
+
+if (!require(devtools)) {
+  install.packages("devtools", repos = c(getOption("repos"), "http://cran.r-project.org"));
+  stopifnot("Could not load `devtools` package." = require(devtools))
+}
```

Comment:
why the swap to devtools here? remotes is the requirement, and devtools depends on it (and is just a lightweight pass thru).

Reply:
to clarify: i'm not sure where devtools dependencies sneak in throughout flepimop, but here its just remotes. if we can eliminate devtools depends elsewhere (e.g. by spinning out the R packages), then we can switch the conda dependency to just remotes, which is like ~20x reduction in dependencies (if they aren't also required by other elements).

Reply:
I think this comment is outdated now. Almost all (minus 1, see GH-332) of the R dependency management is handled directly through conda.